manifests/sites/sample-site-du/06_sriov_network_node_policy_placementrule.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps.open-cluster-management.io/v1
 kind: PlacementRule
 metadata:
   name: placement-sample-site-du-sriov-nnode-policy
-  namespace: sample-site-du-subscription
+  namespace: sites-ran-subscriptions
```

Comment:
have you tried that with multiple sites? i wonder if we need one namespace per site, and if we will have problems when we add 2 subscriptions for 2 sites with the same namespace

Reply:
yes, I test it with different clusters profile combination and it works